tests/system/test_dbapi.py

```diff
@@ -344,6 +344,50 @@ def test_DDL_autocommit(shared_instance, dbapi_database):
             op.result()
 
 
+def test_ddl_executemany(shared_instance, dbapi_database):
+    """Check that DDLs in autocommit mode are immediately executed."""
+
+    try:
+        conn = Connection(shared_instance, dbapi_database)
+        want_row = (
+            1,
+            "first-name",
+        )
+        cur = conn.cursor()
+        cur.execute(
+            """
+            CREATE TABLE Singers (
+                SingerId     INT64 NOT NULL,
+                Name    STRING(1024),
+            ) PRIMARY KEY (SingerId)
+        """
+        )
+        cur.executemany(
+            """
+    INSERT INTO Singers (SingerId, Name)
+    VALUES (%s, %s)
+    """,
+            [want_row],
+        )
+        conn.commit()
+
+        # read the resulting data from the database
+        cur.execute("SELECT * FROM Singers")
+        got_rows = cur.fetchall()
+
+        assert got_rows == [want_row]
+
+        cur.close()
+        conn.close()
+
+    finally:
+        # Delete table
+        table = dbapi_database.table("Singers")
+        if table.exists():
+            op = dbapi_database.update_ddl(["DROP TABLE Singers"])
```

Comment:
From a test execution speed perspective, it would be better not to drop the table here. The reason is that each DDL statement that we execute will generate a new schema version, and Cloud Spanner will start throttling DDL statements if we create too many schema versions.

One alternative could be to generate a unique table name during the test execution instead of a static one. Then we can leave the table around, and rely on the test executor to drop the entire database when all tests have been executed.

Reply:
Done